topostats/grains.py

```diff
@@ -158,23 +158,40 @@ def remove_noise(self, image: np.ndarray) -> np.ndarray:
         Returns
         -------
         np.ndarray
-            2D Numpy array of image with objects < absolute_smallest_grain_size removed.
+            2D Numpy array of image with objects < smallest_grain_size removed.
         """
-        LOGGER.info(f"[{self.filename}] : Removing noise (< {self.absolute_smallest_grain_size})")
-        return remove_small_objects(image, min_size=self.absolute_smallest_grain_size)
+        LOGGER.info(
+            f"[{self.filename}] : Removing noise (< {self.smallest_grain_size} nm^2 \
+{self.smallest_grain_size / (self.pixel_to_nm_scaling**2):.2f} px^2)"
+        )
+        return remove_small_objects(
+            image, min_size=self.smallest_grain_size / (self.pixel_to_nm_scaling**2), **kwargs
+        )
 
     def remove_small_objects(self, image: np.array, **kwargs):
-        """Remove small objects."""
+        """Remove small objects from the input image. Threshold determined by the minimum_grain_size variable of the
+        Grains class which is in pixels squared.
+
+        Parameters
+        ----------
+        image: np.ndarray
+            2D Numpy image to remove small objects from.
+        Returns
+        -------
+        np.ndarray
+            2D Numpy array of image with objects < minimum_grain_size removed.
+        """
         # If self.minimum_grain_size is -1, then this means that
         # there were no grains to calculate the minimum grian size from.
         if self.minimum_grain_size != -1:
             small_objects_removed = remove_small_objects(
                 image,
-                min_size=(self.minimum_grain_size * self.pixel_to_nm_scaling),
+                min_size=self.minimum_grain_size,  # minimum_grain_size is in pixels squared
                 **kwargs,
             )
             LOGGER.info(
-                f"[{self.filename}] : Removed small objects (< {self.minimum_grain_size * self.pixel_to_nm_scaling})"
+                f"[{self.filename}] : Removed small objects (< \
+{self.minimum_grain_size} px^2 {self.minimum_grain_size / (self.pixel_to_nm_scaling)**2} nm^2)"
```

Comment:
Minor formatting suggestion, not essential...
```suggestion
{self.minimum_grain_size} px^2 / {self.minimum_grain_size / (self.pixel_to_nm_scaling)**2} nm^2)"
```